### PR TITLE
[AGENT-4337] Add Airflow DAG example Uploading Dataset to AICatalog and Creating Project

### PR DESCRIPTION
**Pull request by oleksandr-saienko**
# This repository is public. Do not put here any private DataRobot or customer's data: code, datasets, model artifacts, .etc.

## Summary
Added Example of Airflow DAG using recently added new operators (UploadDatasetOperator and extended CreateProject Operator). This DAG uploading local file to DataRobot AICatalog as a static dataset, then using this stored dataset as a training data to create a DataRobot Project.

## Rationale
When we adding new Airflow Operators or extending existing, it's very crucial to provide number of usage examples to make learning curve as easy as possible

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,20 +71,52 @@ in the `context["params"]` variable, e.g. getting a training data you would use
...
+
+          training_dataset_id: str - dataset_id corresponding to existing dataset in DataRobot AICatalog
+          project_name: str - project name
+
+      For more [project settings](https://datarobot-public-api-client.readthedocs-hosted.com/en/latest-release/autodoc/api_reference.html#project) see the DataRobot docs.
+
+      Returns a project ID.
 
-    In case of an S3 input, the `training_data` value must be a [pre-signed AWS S3 URL](https://docs.aws.amazon.com/AmazonS3/latest/userguide/using-presigned-url.html).
+   - Creating project from an existing dataset in Datarobot AICatalog, using dataset_id coming from previous operator.
```
Comment (andrius-senulis) [inline README.md:110]: should be `DataRobot`

**Review by andrius-senulis — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -71,20 +71,52 @@ in the `context["params"]` variable, e.g. getting a training data you would use
...
+
+      In case of an S3 input, the `training_data` value must be a [pre-signed AWS S3 URL](https://docs.aws.amazon.com/AmazonS3/latest/userguide/using-presigned-url.html).
+
+      For more [project settings](https://datarobot-public-api-client.readthedocs-hosted.com/en/latest-release/autodoc/api_reference.html#project) see the DataRobot docs.
 
-        training_data: str - pre-signed S3 URL or local path to training dataset
-        project_name: str - project name
+      Returns a project ID.
+  
+   - Creating project from existing dataset in AICatalog, using dataset_id from config file. Required config params:
```
Comment (elatt) [inline README.md:101]:
```suggestion
   - Creating project from existing dataset in AI Catalog, using dataset_id from config file. Required config params:
```

File: README.md [hunk 3]
```diff
@@ -71,20 +71,52 @@ in the `context["params"]` variable, e.g. getting a training data you would use
...
+
+          training_dataset_id: str - dataset_id corresponding to existing dataset in DataRobot AICatalog
```
Comment (elatt) [inline README.md:103]:
```suggestion
          training_dataset_id: str - dataset_id corresponding to existing dataset in DataRobot AI Catalog
```

File: README.md (hunk 2 above)
Comment (elatt) [reply README.md:101]: Docs us a space for this term.

File: README.md (hunk 1 above)
Comment (elatt) [inline README.md:110]:
```suggestion
   - Creating project from an existing dataset in DataRobot AI Catalog, using dataset_id coming from previous operator. 
```

File: datarobot_provider/example_dags/datarobot_create_project_from_ai_catalog_dag.py [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+from airflow.decorators import dag
+
+from datarobot_provider.operators.ai_catalog import UploadDatasetOperator
+from datarobot_provider.operators.datarobot import CreateProjectOperator
+
+
+@dag(
+    schedule_interval=None,
+    start_date=datetime(2022, 1, 1),
```
Comment (elatt) [inline datarobot_provider/example_dags/datarobot_create_project_from_ai_catalog_dag.py:30]: What is this value for -- should we generate it dynamically?

**Review by elatt — APPROVED** [review]
Just some minor edits with the readme

File: datarobot_provider/example_dags/datarobot_create_project_from_ai_catalog_dag.py (hunk 4 above)
Comment (oleksandr-saienko) [reply datarobot_provider/example_dags/datarobot_create_project_from_ai_catalog_dag.py:30]: this value will be changed by customer when they will reuse this example, so its more like placeholder for DAG execution start date

**Pull request merged**